python/cuopt/cuopt/linear_programming/problem.py

```
@@ -820,6 +1103,9 @@ def _to_data_model(self):
         dm.set_row_types(np.array(self.row_sense, dtype="S1"))
         dm.set_objective_coefficients(self.objective)
         dm.set_objective_offset(self.ObjConstant)
+        if self.ObjQmatrix:
+            Qcsr = self.getQcsr(self.ObjQmatrix)
```

Flake8 (7.3.0)

[error] local variable 'Qcsr' is assigned to but never used

(F841)